torch/_dynamo/backends/tvm.py

```diff
@@ -149,7 +150,7 @@ def to_tvm_tensor(torch_tensor):
     def exec_tvm(*i_args):
         args = [a.contiguous() for a in i_args]
         shape_info, _ = m.get_input_info()
-        active_inputs = {name for name, _ in shape_info.items()}
+        active_inputs = OrderedSet([name for name, _ in shape_info.items()])
```

Comment:
```suggestion
        active_inputs = OrderedSet((name for name, _ in shape_info.items()))
``` 
Does it really need a temp list?

Reply:
Absolutely it does not. :-D

Fixing all of these right now.  (these edits were performed automatically, you see...)

Reply:
@Skylion007: done.

Note that this is a draft - it's really not clear that Dynamo suffers from the same issues that Inductor has with plain old `set` but given this linter/fixer tool, I might as well try it out!

Reply:
Cool, just wanted to make sure I didn't have to update OrderedSet's typing.